### PR TITLE
Automated cherry pick of #1045: Update cent OS repo

### PR DESCRIPTION
**Pull request by doublek**
Cherry pick of #1045 on release-v3.19.

#1045: Update cent OS repo

### CODE REVIEW

**Review by caseydavenport — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
